adal/tests/Test.ADAL.NET.UIAutomation/ADALMobileTestHelper.cs

```diff
@@ -39,13 +39,15 @@ namespace Test.ADAL.UIAutomation
     /// <summary>
     /// Contains the core test functionality that will be used by Android and iOS tests
     /// </summary>
-	public static class ADALMobileTestHelper
+	public class ADALMobileTestHelper
     {
+        CoreMobileTestHelper CoreMobileTestHelper = new CoreMobileTestHelper();
```

Comment:
I believe by .netcore convention the field name should be starting with _c and also it should be marked as private